graphics/QtPBFImagePlugin/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+github.setup        tumic0 QtPBFImagePlugin 1.1
+categories          graphics
+platforms           darwin
+license             LGPL-3
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         PBF image plugin for Qt5
+long_description    Qt image plugin for displaying Mapbox vector tiles.
+
+homepage            https://github.com/tumic0/QtPBFImagePlugin
+
+checksums           rmd160  814697cf675470d17989a0c47c58e10d5879cdf1 \
+                    sha256  7b78877e12a0ca1d8d934333251196b507d2e942a71beb86bd13ac0a2a31c9d7 \
+                    size    193414
+
+configure.args-append PROTOBUF=${prefix} ZLIB=${prefix}
```

Comment:
The spacing here does not conform to the modeline, either add 2 spaces or use a `\` and continue on the next line and align it there.

Reply:
Fixed.